file-formats/intf/zif_aff_intf_v1.intf.abap

```diff
@@ -1,81 +1,85 @@
 INTERFACE zif_aff_intf_v1 PUBLIC.
 
-  TYPES:
-    category_base TYPE n LENGTH 2,
+  "! Interface category
```

Comment:
Should "Interface category" be the description in the json schema for the type ty_category?
Another option would be to use annotation "$description Interface category"

To give a title to the type you have to write:
\<p class="shorttext synchronized" lang="en"\>This is the title\</p>
or
\<p class="shorttext"\>This is the title\</p>
or
$title This is the title

To make a field required in the schema you have to write:
$required

Reply:
Thanks, Nicolas. Let us add these additions as soon as they are clear from your end in an additional change request so that we can merge this one.